tests/unit/components/gallery/HDGalleryMedia.spec.js

```diff
@@ -22,21 +24,45 @@ describe('HdGalleryMedia', () => {
     expect(wrapper.html()).toMatchSnapshot();
   });
 
-  it('hides the thumbnail when the image is loaded', async () => {
-    const { wrapper, img, thumbnail } = build();
+  it('hides the thumbnail when is not provided', async () => {
+    const itemWithoutThumb = { ...ITEMS[0], thumbnail: null };
+    const { thumbnail } = build({ item: itemWithoutThumb });
+
+    expect(thumbnail().exists()).toBe(false);
+  });
+
+  it('checks the thumbnail when an item change', async () => {
```

Comment:
Maybe this title? I was a bit confused with the previous one. Feel free to change it to something else as well.

```suggestion
  it('shows thumbnail when item changes', async () => {
```